tools/code_format/check_format.py

```diff
@@ -914,6 +927,9 @@ def checkErrorMessages(error_messages):
       "./bazel/BUILD",
       "./tools/clang_tools",
   ]
+  bazel_tools_check_excluded_paths = args.bazel_tools_check_excluded_paths + [
+      "./test/",
```

Comment:
Shouldn't this only exclude `test/extensions/common/wasm/test_data/test_cpp/`?

Reply:
Done.